toolchain/fortify-headers/Makefile

```diff
@@ -11,9 +11,9 @@ PKG_NAME:=fortify-headers
 PKG_VERSION:=1.1
 PKG_RELEASE=3
 
-PKG_SOURCE_URL:=https://dl.2f30.org/releases
+PKG_SOURCE_URL:=https://codeload.github.com/jvoisin/$(PKG_NAME)/tar.gz/refs/tags/$(PKG_VERSION)?
```

Comment:
I am not a fan of using these autogenerated GH releases at all.
Also, simply using tags does not guarantee reproducibility as tags can be changed upstream.

So, if you want to use the new upstream, I would suggest pointing to a specific hash instead and letting the build system package the tarball instead.